src/types.ts

```diff
@@ -85,6 +85,7 @@ export interface GameStatus {
 export interface InstallProgress {
   bytes: string
   eta: string
+  folder?: string
```

Comment:
Please keep this consistent across frontend and backend.

Reply:
You mean to add the folder to the BE part? but we don't need it. its dead code from the beginning. I'm using this only to check if the user is installing on the same folder, so the frontend can continue the percentage.
In the future, we could do the other way, and recalculate the percent on the BE, but for that, Electron should support storage on the backend or would need to use a file for that. Which I don't think is that good.

Reply:
I know we don't need it anywhere in the backend. Probably wouldn't as well. This is just for maintaining consistency since one check I think we should have is that frontend and backend types have the same definitions.

Reply:
> I know we don't need it anywhere in the backend. Probably wouldn't as well. This is just for maintaining consistency since one check I think we should have is that frontend and backend types have the same definitions.

But that's another thing and then we should find a way to properly share the same types across both.
We can think about this later but for this PR I don't think its necessary

Reply:
Fixed as well, Mr. consistency 👍🏽